backend/src/services/conflictDetectionService.ts

```
@@ -577,35 +624,46 @@
     duration: number,
     maxAlternatives: number
   ): Promise<{ date: string; startTime: string; endTime: string }[]> {
-    const alternatives: { date: string; startTime: string; endTime: string }[] = [];
-    
+    const alternatives: { date: string; startTime: string; endTime: string }[] =
+      [];
+
     // Search for alternatives in the next 7 days
     for (let i = 0; i < 7 && alternatives.length < maxAlternatives; i++) {
       const searchDate = addDays(preferredDate, i);
       const dayOfWeek = this.getDayOfWeek(searchDate);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)